apps/explorer/src/app/components/render-fetched/index.tsx

```diff
@@ -19,7 +19,9 @@ export const RenderFetched = ({
   }
```

Comment:
Sorry we really should have done this earlier and I apologise, but we need some unit tests for the completed components, can you add tests for the ones in this PR please and thank you 🙏